starter/script.js

```diff
@@ -1 +1,56 @@
-// put code here!
+//=== Program wide functions ===
+var badResponse = function() {
+  console.log("You put in something besides a correct response, try again.");
+};
+var response = 0;
```

Comment:
Do you need to initialize this as 0? You can initialize when saving the value of the initial prompt on Line 9.